package.json

```diff
@@ -34,8 +34,10 @@
   "dependencies": {
     "cors": "^2.8.5",
     "dotenv": "^8.0.0",
+    "es6-promise": "^4.2.8",
```

Comment:
Why do we need this.

Reply:
@domoritz I needed a package called `isomorphic-fetch` for calling the GitHub API. [Their documentation](https://www.npmjs.com/package/isomorphic-fetch) instructed to install this package too.